### PR TITLE
Adding Tresorit.

### PR DESCRIPTION
**Pull request by DimitriDR**
**Type of pull request:** product addition

**Related issues:** N/A

---

Hey folks,
My first contribution here. I give you a new addition to the family of cloud storage.

### CODE REVIEW

File: products/tresorit.toml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+description = "Tresorit is an online cloud storage service based in Switzerland and Hungary."
+slug = "tresorit"
+hostnames = [ "tresorit.com" ]
+sources = [ "https://tresorit.com/legal/privacy-policy", "https://tresorit.com/security", "https://tresorit.com/gdpr"]
+contributors = [ "DimitriDR" ]
+
+[rubric.behavioral-marketing]
+value = "yes-opt-out"
+citations = [
+    "[...] we may use your email address and phone number provided to us to send you marketing communications, [...], **unless you have opted-out**."
```
Comment (doamatto) [inline products/tresorit.toml:11]: Should also mention use of web stats and logs

> We may also use information about you, including web statistics and logs, to personalize the content and experience you receive on our websites or in our marketing communications, as well as by displaying Tresorit ads on other companies' websites and applications, such as on platforms like Facebook and Google. Where legally required, also seek your consent for sending marketing communications.

File: products/tresorit.toml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+[rubric.data-collection-reasoning]
+value = "yes"
+notes = [
+    "See the **How do we use your data?** section in the privacy policy."
+]
+
+[rubric.data-deletion]
+value = "yes-automated"
+notes = [
```
Comment (doamatto) [inline products/tresorit.toml:28]: I don't see mention of this is the privacy policy. I do see it can be done by emailing them (`delete your personal data.`--`You can request this by send an email to support@tresorit.com.`)

File: products/tresorit.toml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    "Option is easily available in account settings."
+]
+
+[rubric.history]
+value = "yes"
+notes = [
+    "At the bottom of the privacy policy, you can observe the archives."
```
Comment (doamatto) [inline products/tresorit.toml:35]: Needs some rephrasing. Seems a bit wordy as is.

File: products/tresorit.toml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+]
+
+[rubric.law-enforcement]
+value = "always"
+citations = [
+    "Tresorit may transmit personal data if the applicable legal provisions so require, or when such action is necessary to comply with any laws, including to meet national security or law enforcement requirements."
```
Comment (doamatto) [inline products/tresorit.toml:41]: More aptly `reasonable` as it's as necessary, but not `strict` as it isn't exclusively given in court order, subpoena, et al.

File: products/tresorit.toml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+[rubric.list-collected]
+value = "generally"
+notes = [
+    "See the **What kind of personal data do we process?** section in the privacy policy."
+]
+
+[rubric.noncritical-purposes]
+value = "no"
+notes = [
+    "No mean."
```
Comment (doamatto) [inline products/tresorit.toml:53]: No note needed.

File: products/tresorit.toml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+]
+
+[rubric.list-collected]
+value = "generally"
+notes = [
+    "See the **What kind of personal data do we process?** section in the privacy policy."
+]
+
+[rubric.noncritical-purposes]
+value = "no"
```
Comment (doamatto) [inline products/tresorit.toml:51]: Could argue Marketing, Aggregate Insights, and "Other purposes" are all non-critical uses.

File: products/tresorit.toml [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    "If there are any material changes to this policy, you will be notified 30 days prior to the change becoming effective."
+]
+
+[rubric.security]
+value = "yes-independent-audits"
+citations = [
+    "We take appropriate technical and organizational measures to protect your personal data against loss or other forms of unlawful processing. Tresorit is ISO 27001:2013 certified."
+]
+notes = [
+    "ISO controls annually"
```
Comment (doamatto) [inline products/tresorit.toml:68]: This comment doesn't add anything of importance and can be removed

File: products/tresorit.toml [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+[rubric.revision-notify]
+value = "yes"
+citations = [
+    "If there are any material changes to this policy, you will be notified 30 days prior to the change becoming effective."
+]
+
+[rubric.security]
+value = "yes-independent-audits"
+citations = [
+    "We take appropriate technical and organizational measures to protect your personal data against loss or other forms of unlawful processing. Tresorit is ISO 27001:2013 certified."
```
Comment (doamatto) [inline products/tresorit.toml:65]: Should note who audits and where to read more about the audits

File: products/tresorit.toml [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+]
+notes = [
+    "ISO controls annually"
+]
+
+[rubric.third-party-access]
+value = "yes-specified-noncritical"
+citations = [
+    "In certain cases we need to share information, including personal data with our third-party service providers. We use third-party service providers for a number of services, including application development, backup, storage, payment processing, analytics and other services."
```
Comment (doamatto) [inline products/tresorit.toml:74]: There is a list of providers mentioned a few sentences later: https://support.tresorit.com/hc/en-us/articles/216114397-Third-party-services

**Review by doamatto — CHANGES_REQUESTED** [review]
Good start; needs some more work however before it can be merged.

File: products/tresorit.toml (hunk 2 above)
Comment (DimitriDR) [reply products/tresorit.toml:28]: Actually, you can do it both ways: https://support.tresorit.com/hc/en-us/articles/360008033453-How-to-delete-your-account or by sending e-mail.

Comment (doamatto) [reply products/tresorit.toml:28]: In that case, I'd make a note to that page explaining that it can be used for self-service data deletion, but still also cite that you can request deletion over email.

**Comment by milesmcc** [comment]
Thanks @doamatto for all the comments.

File: products/tresorit.toml (hunk 6 above)
Comment (doamatto) [reply products/tresorit.toml:55]: Could still be argued that marketing and "other purposes" are non-critical.

File: products/tresorit.toml [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+    "If there are any material changes to this policy, you will be notified 30 days prior to the change becoming effective."
+]
+
+[rubric.security]
+value = "yes-independent-audits"
+citations = [
+    "We take appropriate technical and organizational measures to protect your personal data against loss or other forms of unlawful processing. Tresorit is ISO 27001:2013 certified."
+]
+notes = [
+    "TÜV Rheinland Group has made the ISO certification."
```
Comment (doamatto) [inline products/tresorit.toml:69]: A link to the audit or where to find the audit might not be a bad idea

**Review by doamatto — CHANGES_REQUESTED** [review]

File: products/tresorit.toml (hunk 6 above)
Comment (DimitriDR) [reply products/tresorit.toml:55]: I don't understand what do you mean. What do you want me to write?
